lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -748,7 +750,8 @@ instance Arbitrary PostExternalTransactionData where
     arbitrary = do
         count <- choose (0, 32)
         bytes <- BS.pack <$> replicateM count arbitrary
-        return $ PostExternalTransactionData bytes
+        let hex = convertToBase Base16 bytes :: B8.ByteString
```

Comment:
Should we remove this type annotation?

Reply:
indeed! removed the annotation

Reply:
I think the `newtype PostExternalTransactionData` wrapper can be removed altogether from the API and just replaced with ByteString.

In the CLI, you don't need `FromText PostExternalTransactionData`, just use [`eitherReader`](http://hackage.haskell.org/package/optparse-applicative-0.15.0.0/docs/Options-Applicative.html#v:eitherReader):

```haskell
argument (eitherReader (bimap show S8.unpack . convertFromBase Base16 . S8.pack))
```